### PR TITLE
Adding my name function

### PR DESCRIPTION
**Pull request by CharlieAdisa**
Files I changed:
HelloPullRequest.py I changed the name function:)

### CODE REVIEW

File: HelloPullRequest.py [hunk 1]
```diff
@@ -1 +1,4 @@
-# And a function that print your name ete SabastianMugazambi({String}): print String
\ No newline at end of file
+# And a function that print your name ete SabastianMugazambi({String}): print String
+
+def charlieAnderson(name):
+	print name
```
Comment (ammaradisa) [inline HelloPullRequest.py]: this is a comment

**Review by ammaradisa — APPROVED** [review]
I'm a nice guy

File: HelloPullRequest.py (hunk 1 above)
Comment (CharlieAdisa) [reply HelloPullRequest.py]: noice

Comment (SabastianMugazambi) [reply HelloPullRequest.py]: write a better comment.

**Review by ammaradisa — COMMENTED** [review]
leave Charlie alone

File: HelloPullRequest.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# And a function that print your name ete SabastianMugazambi({String}): print String
+# And a function that print your name ete SabastianMugazambi({String}): print String 
 
 def charlieAnderson(name):
-	print name
+	print(name)
```
Comment (CharlieAdisa) [inline HelloPullRequest.py]: k

**Pull request merged**